python/setup.py

```diff
@@ -83,7 +83,10 @@ def wrapper():
         Extension("_fastrpc", [
             "fastrpcmodule.cc", "pythonserver.cc", "pyerrors.cc",
             "pythonbuilder.cc", "pythonfeeder.cc"
-        ], libraries=["fastrpc"]),
+        ],
+                  libraries=['xml2'],
```

Comment:
Bad code alignment?

Reply:
No, pycharm allign it with `Extension(`. I try to make it more readable.